app/views/members/_form.html.erb

```diff
@@ -39,12 +38,7 @@
     <%= f.text_area :bio %>
   </div>
 
-  <p style="width: 1px; height: 1px; overflow: hidden;">
-    Please leave following field blank:
-    <%= f.text_area :username %>
```

Comment:
Thanks @zenspider, I was stuck on redirecting to root in the controller if a spam bot fills in username.  Looks like we can override create and call super.

http://stackoverflow.com/questions/4783604/overriding-devises-registration-controller-to-allow-for-a-redirect-after-a-succ